### ISSUE TITLE
recurrence of issue #211 Password Protected blogs don't work anymore, instead giving a 'you need to enter a password' error, even though it's already entered

### ISSUE THREAD

**Issue by Kvothe1970** [johanneszab/TumblThree#221]
Is it possible that this is an recurrence of issue #211 ? After this fix was released, my one password protected blog worked just fine. Now it suddenly acts up again. Other blogs work fine. I triple checked the password and can log in fine with the browser. I have set up a blog with a few of my private food pics and made it password protected. 

Test in Version 1.0.8.46 
Same error. Is anyone else experiencing this again? 

Blog is https://culinaryreactions.tumblr.com/ 
pw is culinaryreactions

**Comment by johanneszab** [comment]
No, it's not the exact same issue. I haven't changed the code, but apparently Tumblr doesn't allow to view password protected blogs using the old api (v1) anymore, even if the request uses the "Basic Authentication"/WWW-Authenticate-Header which is what we're currently doing.

However, I've tested it using the svc code (another, unofficial tumblr api that is used for the dash board blogs) that handles the hidden tumblr crawls, and it looks like it's already capable of crawling password protected blogs. As a test, after closing TumblThree, I've renamed the extension from .tumblr to .tmblrpriv of the index files and changed the [BlogType](https://github.com/johanneszab/TumblThree/blob/master/src/TumblThree/TumblThree.Domain/Models/BlogTypes.cs) from 0 to 1 and that already fixed the issue.

So all that has to be done is changing the [TumblrBlogDetector.cs](https://github.com/johanneszab/TumblThree/blob/master/src/TumblThree/TumblThree.Applications/Crawler/TumblrBlogDetector.cs) class so that it checks if a blog uses a password, and if it needs one, the TumblrHiddenCrawler.cs has to be used instead of the TumblrBlogCrawler.cs.

**Comment by johanneszab** [comment]
Thanks for sharing!

I'll probably fix this in the next days.

**Comment by Kvothe1970** [comment]
Cheers for reacting so quickly. I'll just the workaround described by you earlier for now until you had a chance to fix this.

**Comment by Kvothe1970** [comment]
Which didn't work. I did as outlined in @ #185 and the blog always ends up marked as "offline" 
Just reverted the changes in the files and the extensions and will wait patiently for the fix )

**Comment by Hrxn** [comment]
> Blog is https://culinaryreactions.tumblr.com/
pw is culinaryreactions

I just see an empty blog.. ?

**Comment by Kvothe1970** [comment]
Just tested here at work with a password protected blog. Works like a charm. No error messages either.

**Comment by Kvothe1970** [comment]
Update: Tested it at home with my full list. Now the number of blogs reporting "Post was not parseable, set to 1" has either increased, or 50% of my blogs have posts like that over night. Can anyone else please test this?

**Comment by Kvothe1970** [comment]
Unbelievable. Tumblr staff makes me wonder with every single bloody update. Thanks for double checking this Johannes.

**Comment by EC-O-DE** [comment]
oh I hope they don't f*uck th*mes
